content/building/build-nxp.md

```diff
@@ -62,7 +62,7 @@ The setup is a lot easier than it seems. The setup scripts do almost everything.
 
 If you intend to change the nanoBooter or nanoCLR and create Pull Requests then you will need to fork the [nanoFramework/nf-interpreter](https://github.com/nanoFramework/nf-interpreter) to your own GitHub repo and clone the forked GitHub repo to your Windows system using an Git client such as the [GitHub Desktop application](https://desktop.github.com/).
 
-The _develop_ branch is the default working branch. When working on a fix or experimenting a new feature you should do it on another branch. See the [Contributing guide](..\..\content\contributing\contributing-workflow.md#suggested-workflow) for specific instructions on the suggested contributing workflow.
+The _develop_ branch is the default working branch. When working on a fix or experimenting a new feature you should do it on another branch. See the [Contributing guide](https://github.com/nanoframework/nanoframework.github.io/tree/pages-source/content/contributing) for specific instructions on the suggested contributing workflow.
```

Comment:
It would be best to use a relative link here (otherwise it is unlikely to show properly on the actual website.
Also, please feel free to update other instances where this link is wrong in other documents

Reply:
When I say reletive link it probably had too little (or to many) `../` tags